proto/tendermint/abci/types.proto

```diff
@@ -212,6 +233,12 @@ message ResponseCheckTx {
   repeated Event events     = 7
       [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
   string codespace = 8;
+  string sender    = 9;
+  int64  priority  = 10;
+
+  // mempool_error is set by Tendermint.
+  // ABCI applictions creating a ResponseCheckTX should not set mempool_error.
+  string mempool_error = 11;
```

Comment:
Is this update from upstream Tendermint?

Reply:
yes, not from me

Reply:
Is there a way to get the changes from upstream tendermint into a separate PR. Will reduce the diff and make this PR easier to review.

Reply:
After rebasing, this looks as it should :+1: